src/cirq_iqm/iqm_remote.py

```diff
@@ -93,17 +93,17 @@ def _send_circuit(
             circuit: cirq.Circuit,
             repetitions: int = 1,
     ) -> cirq.study.Result:
-        """Sends the circuit to be executed by the remote IQM device.
+        """Sends the circuit to be executed on the IQM quantum computer.
```

Comment:
```suggestion
        """Sends the circuit to be executed by the IQM computation service.
```

Reply:
I'll remove the controversial part. It reads just as well like that :) Underscore method, after all.